File: src/cpp/number-9.cpp
#include <iostream>
#include <random>

int main() {
    srand(time(NULL));

    if (rand() % 9 + 1 == 9) {
        std::cout << "Number 9...\n";
    } else {
        std::cout << "Try again.\n";
    }

    return 0;
}

File: src/go/number-9.go
package main

import (
    "fmt"
    "math/rand"
)

func main() {
    number := rand.Uint32() % 10

    if number == 9 {
        fmt.Println("Number 9...")
    } else {
    	fmt.Println("Try again.")
    }
}
